src/net_util.h

```diff
@@ -261,6 +261,15 @@ inline double htond(double d)
 	return d;
 	}
 
+inline float ntohf(float f)
+	{
+	return d;
```

Comment:
Shouldn't this one and the next reference `f` rather than `d`?

Reply:
Yes 🤦🏼